Estilos css

oscarjch10 opened the pull request:
Cambio en html y Css

oscarjch10 committed:
Estilos css

Cambio en html y Css

paulchino34 commented on third-project/css/estilos.css:
```diff
@@ -0,0 +1,166 @@
+		background: url(https://img-s1.onedio.com/id-533c1665587bd3c20e009960/rev-0/w-620/f-jpg/s-504b348813175db21107e009db9cb291e75b47bb.jpg);
+		background-size: 100vw 100vh;
+		background-repeat: no-repeat;
+        
```
retira espacios en blanco innecesarios

paulchino34 commented on third-project/css/estilos.css:
```diff
@@ -0,0 +1,166 @@
+        
+        
+	}
+	.capa{
```
utiliza ingles para los nombres de variables utiliza (layer)

paulchino34 commented on third-project/css/estilos.css:
```diff
@@ -0,0 +1,166 @@
+	/*Menù lateral*/
+	#btn-menu{
+		display: none;
+	}
```
utiliza un espacio luego de cerrar un bloque

paulchino34 replied:
esto aplica para todo el documento

paulchino34 commented on third-project/css/estilos.css:
```diff
@@ -0,0 +1,166 @@
+		opacity: 1;
+		visibility: visible;
+	}
+	.cont-menu{
```
utiliza nombre mas dicientes o que permitan interpretar lo que se hace referencia

Luisfer082 committed:
responsive

responsive hasta 850px, se agrego footer,  la imagen no se deforma, falta responsive por debajo de 850px

Luisfer082 committed:
Arreglos css y resposive

Se optimizo el codigo para poder hacer responsiva la pagina. Se acomodaron las graficas
